modules/rsvp/client/components/rsvp-form.component.ts

```diff
@@ -74,7 +74,7 @@ export class RsvpFormComponent implements OnInit{
         console.log(data);
         this.NotificationsService.success("Thank you for submitting your rsvp!");
         setTimeout(()=> {
-          this.router.navigate(['/']);
+          window.location.href="/";
```

Comment:
this was the only way I could get a full page reload. I tried router.navigateByUrl and router.navigate, neither fully loaded. I want a full refresh because the logic for this is simpler than using an event